src/styles/views/Lobby.scss

```diff
@@ -0,0 +1,106 @@
+@import "../theme";
+
+/* The '&' is a shorthand for the containing element,
+  so for example a '&.user-list' inside .game will compile to
+  .game.user-list
+
+  for more information visit https://sass-lang.com/guide
+ */
+
+.lobbypage {
+
+  &.container {
+    float: left;
+    background: $background;
+    padding: 1.5em;
+    border-radius: $borderRadius;
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    box-shadow: $dropShadow;
+    margin-left: 30px;
+  }
+
+  & p,
+  h2 {
+    margin: 0.5em 0;
+  }
+}
+
+.gamemode {
+
+  &:hover {transform: translateY(-2px);}
+  &.container {
+    text-align: start;
+    margin: 0.5em 0;
+    width: auto;
+    padding: 20px;
+    border-radius: $borderRadius;
+    display: flex;
+    align-items: center;
+    background: lighten($background, 20);
+    cursor: pointer;
+
+    &.selected {
+      background: lighten($background, 25);
+      border: solid rgb(0, 0, 0)
+    }
+  }
+
+    &.name {
+    text-align: center;
+    margin: auto;
+    color: $accentDark;
+    text-transform: uppercase;
+  }
+
+  &.description {
+    color: $textColor;
+
+  }
+
+  &.list {
+    list-style: none;
+    text-align: center;
+  }
+
+}
+
+.player {
+
+  &.container {
+    float: left;
+    background: lighten($background, 20);
+    padding: 20px;
+    border-radius: $borderRadius;
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    width: 200px;
+  }
+
+  &.list {
+    list-style: none;
+    padding-left: 0;
+  }
+
+  &.icon {
+    position: relative;
+    top: 10%;
+    max-width: 150px;
```

Comment:
Change `max-width: 150px` to `max-width: 80px`
Change `max-height: 150px` to `max-height: 80px`

This is still debatable but in a player overview, I don't think profile icons should take up too much space.